assets/js/index.js

```diff
@@ -0,0 +1,259 @@
+              </div>
+            </div>
+          </div>`;
+  weatherInfoContainer.append(currentWeatherCard);
+};
+
+const renderForecastData = (data) => {
+  console.log("data: " + data);
+
+  const createForecastCard = (each) => {
+    const forecast = `<div class="card" style="width: 12rem">
+      <img
+        class="card-img-top card-image"
+        src="http://openweathermap.org/img/w/${each.weather[0].icon}.png"
+        alt="Card image cap"
+      />
+
+      <div class="card-body">
+        <h5 class="card-title">${moment.unix(each.dt).format("ddd, Do MMM")}
+         </h5>
+        <p class="card-text">Temp: ${each.temp.day}</p>
+        <p class="card-text">Wind: ${each.wind_speed}</p>
+        <p class="card-text">Humidity:${each.humidity}</p>
+      </div>
+    </div>`;
+    return forecast;
+  };
+
+  const forecastCard = data.weatherData.daily
+    .slice(1, 6)
+    .map(createForecastCard)
+    .join("");
+
+  const forecastWeatherCard = `<div>
+            <h4 class="my-3 fw-bold">5-day Forecast:</h4>
+            <div class="d-flex flex-wrap justify-content-between">
+             ${forecastCard}
+            </div>
+            <!--closing div for flex class-->
+          </div>`;
+
+  weatherInfoContainer.append(forecastWeatherCard);
+};
+
+const renderErrorAlert = () => {
+  weatherInfoContainer.empty();
+
+  const alert = `<div class="alert alert-info" role="alert" id = "alert-ul">
+         Please enter the city name.
+        </div>`;
+  weatherInfoContainer.append(alert);
+};
+
+const rendertWeatherInfo = async (cityName) => {
+  try {
+    const weatherData = await renderWeatherData(cityName);
+    weatherInfoContainer.empty();
+
+    // render current weather data and append to the section
+    renderCurrentData(weatherData);
+
+    // render forecast data via jquery
+    renderForecastData(weatherData);
+  } catch (error) {
+    renderErrorAlert();
+  }
+};
+
+const renderForecastWeather = (forecastWeatherData) => {
+  // render forecast weather data and append  each card to the section
+};
+
+const handleFormSubmit = async (event) => {
+  event.preventDefault();
+
+  // get city name from input
+  const cityName = $("#city-name").val();
+  console.log("You entered " + cityName);
+  // if  not empty
+  if (cityName) {
+    await rendertWeatherInfo(cityName);
+
+    const recentSearches = getFromLocalStorage("recentSearches", []);
+
+    if (!recentSearches.includes(cityName)) {
+      recentSearches.push(cityName);
+    }
+    writeToLocalStorage("recentSearches", recentSearches);
+    // remove last section
+    removeAlert.remove();
+    //recentSearchContainer.children().last().remove();
+
+    // rerender recent cities
+    renderCities();
+  }
+  // render weather data
+};
+
+const handleSearchClick = (event) => {
+  const target = $(event.target);
+
+  if (target.is("li")) {
+    const cityName = target.attr("data-city");
```

Comment:
Seems like you are missing the function call to re-render the weather information with the city name and this is a missing requirement